jscomp/others/js_json.res

```diff
@@ -30,6 +30,7 @@ type rec t =
   | @as(null) Null
   | String(string)
   | Number(float)
+  | Bigint(bigint)
```

Comment:
BigInt doesn't belong to JSON spec

Reply:
Ah, correct. I'll remove it, thanks!

Reply:
Removed https://github.com/rescript-lang/rescript-compiler/commit/c8b78d26d225c06e2d04be9c6ba5f4837259e638